common/src/types/v0/store/pool.rs

```diff
@@ -69,6 +67,9 @@ impl PartialEq<CreatePool> for PoolSpec {
     }
 }
 
+// PoolLabel is the type for the labels
+type PoolLabel = ::std::collections::HashMap<String, String>;
```

Comment:
any particular reason to redefine PoolLabel again? You can mark the other one as pub and then you can use it `message_bus::PoolLabel` or something like that

Reply:
No reason, I removed it and added back at a different place. I will make it pub. Thanks